retrofit/src/test/java/retrofit2/RequestBuilderTest.java

```diff
@@ -2578,6 +2579,84 @@
     assertThat(readBody.indexOf("secondParam")).isLessThan(readBody.indexOf("thirdParam"));
   }
 
+  @Test public void queryParamsSkippedIfConvertedToNull() throws Exception {
+    class Example {
+      @GET("/query") Call<ResponseBody> queryPath(@Query("a") Object a) {
+        return null;
```

Comment:
these could probably throw AssertionErrors, but all the other tests return null, too, so 👍 consistency.